src/main/java/school/lemon/changerequest/java/introduction/hw1/Task6.java

```diff
@@ -1,11 +1,25 @@
 package school.lemon.changerequest.java.introduction.hw1;
 
 public class Task6 {
+
+    private static int factorial(int n) {
+        if (n == 0) return 1;
+        return n * factorial(n - 1);
+           }
+
     public static double calculateS(double x) {
-        return 0d;
+        double s;
+        s = 1 + x;
+        s += (Math.pow(x,2))/factorial(2);
+        s+= (Math.pow(x,3))/factorial(3);
+        s+=(Math.pow(x,4))/factorial(4);
```

Comment:
Assume that you need to calculate 10 more terms, will you add 10 more `s+=` statements?
Try to solve this task using any loop.